utils/fake_libc_include/zlib.h

```diff
@@ -1,2 +1,33 @@
+#ifndef ZLIB_H
+#define ZLIB_H
+
 #include "_fake_defines.h"
 #include "_fake_typedefs.h"
+
+typedef int uInt
+typedef int uLong
+#if !defined(__MACTYPES__)
+typedef int Byte
+#endif
+
+typedef int Bytef
+typedef int charf
+typedef inf intf
```

Comment:
Is this really `inf`?

Reply:
Arrgh. Sorry about that, I must have been asleep when doing this. You missed all the missing semi-colons ;-)

My test runs included the standard `fake_lib` instead, which masked the updated one. With https://github.com/eliben/pycparser/pull/368/commits/9557a944b4bac002b6a1fb6532c79a25ba715a5e it compiles with `gcc` 7.4, except for a duplication of `__built_in_va_list` which is not related to `zlib.h`:

```
e$ gcc zlib.h
In file included from zlib.h:5:0:
_fake_typedefs.h:5:13: error: conflicting types for ‘__builtin_va_list’
 typedef int __builtin_va_list;
             ^~~~~~~~~~~~~~~~~
cc1: note: previous declaration of ‘__builtin_va_list’ was here
```